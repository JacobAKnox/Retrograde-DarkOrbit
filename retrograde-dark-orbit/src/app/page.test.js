import { render } from '@testing-library/react'
const { default: Home } = require("./page");
import "setimmediate";

it('renders main page unchanged', () => {
    const page = render(<Home/>);
    expect(page).toMatchSnapshot();
});