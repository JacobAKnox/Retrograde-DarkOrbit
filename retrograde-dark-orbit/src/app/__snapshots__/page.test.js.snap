// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders main page unchanged 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="flex justify-center items-start"
      >
        <div
          class="flex w-1/2 min-h-screen flex-row items-center justify-between p-24"
        >
          <div>
            <h1
              class="text-center text-2xl text-blue-300"
            >
              <b>
                JOIN GAME
              </b>
            </h1>
            <div
              class="bg-inherit p-4 m-4 items-center flex flex-col"
            >
              <input
                aria-label="Username"
                class="bg-slate-700 text-white m-1 py-2 px-10 rounded-xl hover:bg-slate-600 disabled:bg-slate-900 placeholder:disabled:text-gray-700"
                placeholder="Username"
                value=""
              />
              <input
                aria-label="Code"
                class="bg-slate-700 text-white m-1 py-2 px-10 rounded-xl hover:bg-slate-600 disabled:bg-slate-900 placeholder:disabled:text-gray-700"
                placeholder="Code"
                value=""
              />
              <button
                class="bg-slate-900 text-white m-1 py-2 px-10 rounded-xl hover:bg-slate-800 disabled:bg-slate-950 disabled:text-gray-700"
                disabled=""
              >
                Join
              </button>
              <p
                class="bg-inherit text-red-500 m-1 py-1 px-2 text-center text-lg text-wrap"
              />
            </div>
          </div>
          <div>
            <h1
              class="text-center text-2xl text-blue-300"
            >
              <b>
                CREATE GAME
              </b>
            </h1>
            <div
              class="bg-inherit p-4 m-4 items-center flex flex-col"
            >
              <input
                aria-label="Username"
                class="bg-slate-700 text-white m-1 py-2 px-10 rounded-xl"
                placeholder="Username"
                value=""
              />
              <p
                class="text-red-500 text-xs italic"
              />
              <button
                class="bg-slate-900 text-white m-1 py-2 px-10 rounded-xl"
              >
                Join
              </button>
            </div>
          </div>
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="flex justify-center items-start"
    >
      <div
        class="flex w-1/2 min-h-screen flex-row items-center justify-between p-24"
      >
        <div>
          <h1
            class="text-center text-2xl text-blue-300"
          >
            <b>
              JOIN GAME
            </b>
          </h1>
          <div
            class="bg-inherit p-4 m-4 items-center flex flex-col"
          >
            <input
              aria-label="Username"
              class="bg-slate-700 text-white m-1 py-2 px-10 rounded-xl hover:bg-slate-600 disabled:bg-slate-900 placeholder:disabled:text-gray-700"
              placeholder="Username"
              value=""
            />
            <input
              aria-label="Code"
              class="bg-slate-700 text-white m-1 py-2 px-10 rounded-xl hover:bg-slate-600 disabled:bg-slate-900 placeholder:disabled:text-gray-700"
              placeholder="Code"
              value=""
            />
            <button
              class="bg-slate-900 text-white m-1 py-2 px-10 rounded-xl hover:bg-slate-800 disabled:bg-slate-950 disabled:text-gray-700"
              disabled=""
            >
              Join
            </button>
            <p
              class="bg-inherit text-red-500 m-1 py-1 px-2 text-center text-lg text-wrap"
            />
          </div>
        </div>
        <div>
          <h1
            class="text-center text-2xl text-blue-300"
          >
            <b>
              CREATE GAME
            </b>
          </h1>
          <div
            class="bg-inherit p-4 m-4 items-center flex flex-col"
          >
            <input
              aria-label="Username"
              class="bg-slate-700 text-white m-1 py-2 px-10 rounded-xl"
              placeholder="Username"
              value=""
            />
            <p
              class="text-red-500 text-xs italic"
            />
            <button
              class="bg-slate-900 text-white m-1 py-2 px-10 rounded-xl"
            >
              Join
            </button>
          </div>
        </div>
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
